src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -89,7 +89,9 @@ public function get_event_data( $event_id, $context = '' ) {
 			return new WP_Error( 'event-not-found', $this->messages->get_message( 'event-not-found' ) );
 		}
 
-		$meta = array_map( 'reset', get_post_custom( $event_id ) );
+		$meta = array_map( function ( $item ) {
+			return reset( $item );
```

Comment:
Just a comment - we do this three times, it could be extracted out into a named function. It's simple enough it doesn't really matter though.